apps/web/lib/settings/platform/utils.ts

```diff
@@ -0,0 +1,104 @@
+type IndividualPlatformPlan = {
+  plan: string;
+  description: string;
+  pricing?: number;
+  includes: string[];
+};
+
+type HelpCardInfo = {
+  icon: string;
+  variant: "basic" | "ProfileCard" | "SidebarCard" | null;
+  title: string;
+  description: string;
+  actionButton: {
+    href: string;
+    child: string;
+  };
+};
+
+// if pricing or plans change in future modify this
+export const platformPlans: IndividualPlatformPlan[] = [
```

Comment:
This `platformPlans` variable here is not being used anywhere at the moment, but will be needed later when we remove [platformUtils](https://github.com/calcom/cal.com/blob/33c1d49c521912e9dbb98165613c47709e416b3a/apps/web/components/settings/platform/platformUtils.ts#L4) and use `platformPlans` from this file so that our folder schedule is uniform all across.